### ISSUE TITLE
LearnBox macro not working

### ISSUE THREAD

**Issue by estelle** [mdn/content#15668]
### MDN URL

https://developer.mozilla.org/en-US/docs/Glossary

### What specific section or headline is this issue about?

Glossary

### What information was incorrect, unhelpful, or incomplete?

The LearnBox macro is not working. Users see: {{LearnBox({"title":"Learn a new word ..."})}}

### What did you expect to see?

a glossary term and definition 

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/glossary`
* MDN URL: https://developer.mozilla.org/en-US/docs/Glossary
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/glossary/index.md
* Last commit: https://github.com/mdn/content/commit/21c2c70b67f776e0e57bca14d64d84fd451562bd
* Document last modified: 2022-03-20T23:50:03.000Z

</details>

**Comment by teoli2003** [comment]
`LearnBox` was picking a term each time a page was regenerated. During wiki times it was each time the page was reloaded. It was an easy way to quiz oneself about developer-related concepts.

With yari, a page is generated once a day, so, theoretically, this macro will ask for one single word per day (à la Wordle).

Given that it is of dubious usefulness, broken, and only used in two pages, I propose that we:
- just remove the two occurrences here on mdn/content
- mark it deprecated on mdn/yari and/or delete it from mdn/translated-content.

In other words, let's get rid of this macro.
